jck/runtime.api/playlist.xml

```diff
@@ -931,6 +931,12 @@
 				<impl>ibm</impl>
 				<platform>.*zos.*</platform>
 			</disable>
+			<disable>
+				<comment>Disabled on z/OS due to backlog/issues/744</comment>
+				<version>11</version>
+				<impl>ibm</impl>
+				<platform>.*zos.*</platform>
```

Comment:
Maybe specify s390x_zos and s390_zos?

Reply:
Could you please state exactly what string value you are suggesting for `<platform>` here?  I thought `.*zos.*` was clear enough - it includes all z/OS platforms.

Reply:
Actually, since the only z/OS platform on which we run tck 11 is s390x, having `s390x_zos` for `<platform>` should suffice. I've updated the PR.